_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -1853,17 +1971,20 @@ Create all the corresponding input variables for `account-baseline-app` in `vari
 # ---------------------------------------------------------------------------------------------------------------------
 
 variable "name_prefix" {
-  description = "The name used to prefix AWS Config and CloudTrail resources, including the S3 bucket names and SNS topics used for each."
+  description = "The name used to prefix AWS Config and CloudTrail resources,
+    including the S3 bucket names and SNS topics used for each."
```

Comment:
Is there a new line here? If so, is that valid Terraform syntax? I understand the desire for readability, but if you copy/paste this snippet, will it be broken? Same for the variables below.

Reply:
Agh, you're right. I didn't actually mean to do the formatting of variables/descriptions here. Force of habit.